Using HTML, CSS, Bootstrap, JS

jainamb0907 opened the pull request:
1. **Carousel-using-bootstrap:** Implemented using bootstrap, which has sliders with certain time interval, manual buttons, and captions.
2. **List-with-floating-section-headings**: Implemented using HTML-CSS. I used the concept of description list in HTML and use the concept of pseudo-class. In CSS I used grid, margin, position, border-radius etc.  properties for designing
3. **Toast**: I totally used Bootstrap in this task and customize basic toast using CSS properties.
4. **Multi-colored-text:** These task is done Using linear-gradient attribute in which I used **<span> tag** to display the text and in CSS part I used, 
    - The webkit properties: fill the text with the gradient background 
    - The color property with transparent background
    - CSS background-image property: to add a gradient background to your text
    - The background-clip property: to get multicolor text instead of the multicolor background

5. **Same-styling-on-two-header-tags**:  I used one common class and apply it on every Header tag.

jainamb0907 committed:
Using HTML, CSS, Bootstrap, JS

dilaara-simform commented on Tasks/Toast/index.html:
```diff
@@ -0,0 +1,30 @@
+</head>
+
+<body>
+
```
implement two type of toaster in flipkart application one with success message and one with error. 
one should be closed automatically and one will be closed on user click.

dilaara-simform commented on Tasks/multi-coloured-text/index.html:
```diff
@@ -0,0 +1,29 @@
+    <span class="multicolortext">Jainam D Bhavsar</span>
+  </h1>
+
+  <script src="script.js"></script>
```
what is the use of this file.

dilaara-simform commented on Tasks/multi-coloured-text/index.html:
```diff
@@ -0,0 +1,29 @@
+  <!--   The modern method: Using linear-gradient attribute -->
+
+  <h1>
+    <span class="multicolortext">Jainam D Bhavsar</span>
```
change colour of whole word on hover.

dilaara-simform commented on Tasks/Carousel-using-bootstrap/index.html:
```diff
@@ -0,0 +1,81 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="style.css" rel="stylesheet" type="text/css" />
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous" />
```
images are getting streched so need to modify css

jainamb0907 committed:
Corrections

Carousel-using-bootstrap: solve issue of stretching images, multi-coloured-text: when hover,the color of multicolored-text is changed
